### ISSUE TITLE
SA1516 is reported for the declarator, not the identifer

### ISSUE THREAD

**Issue by pdelvo** [DotNetAnalyzers/StyleCopAnalyzers#558]
If SA1516 should be reported for this code:
```csharp
private static readonly bool foo = false;
```
Then "foo = false" is the location for the diagnostic. I think reporting on just "foo" would be better.

**Comment by sharwell** [comment]
Agreed. In addition, if multiple variable declarators are included, the warning should probably only be reported for the first of them.

Another option is only reporting it for the field *type* and/or modifiers, since those appear before the first variable and only appear once.

**Comment by sharwell** [comment]
Is this superseded by #562?
